README.md

```diff
@@ -558,6 +558,8 @@ const uiSchema = {
 };
 ```
 
+In addition it is valid to mark an array as "required".  This means the user must select at least one item to add to the array.
```

Comment:
I'm OK with your proposed solution, @n1k0. I think it is an improvement to the current situation either way.